tests/Doctrine/Tests/DBAL/Functional/Driver/OCI8/StatementTest.php

```diff
@@ -39,17 +39,41 @@ public function testQueryConversion(string $query, array $params, array $expecte
         );
     }
 
+    /**
+     * Low-level approach to working with parameter binding
+     *
+     * @param mixed[] $params
+     * @param mixed[] $expected
+     *
+     * @dataProvider queryConversionProvider
+     */
+    public function testStatementBindParameters(string $query, array $params, array $expected) : void
+    {
+        $stmt = $this->connection->prepare($query);
+        $stmt->execute($params); // bind params in statement and execute it
```

Comment:
>  // bind params in statement and execute it

This is obvious from the method phpDoc. Please remove.

Reply:
removed